aws-iot-thingtype/src/main/java/software/amazon/iot/thingtype/Translator.java

```diff
@@ -206,4 +185,37 @@ static TagResourceRequest tagResourceRequest(final String arn, final Collection<
                 .resourceArn(arn)
                 .tags(tags).build();
     }
+
+    static software.amazon.iot.thingtype.ThingTypeProperties translateThingTypePropertiesToModelObject(
```

Comment:
do we need the full class path here for ThingTypeProperties ?

Reply:
We have 2 definition of ThingTypeProperties is the code base - 
- software.amazon.awssdk.services.iot.model.ThingTypeProperties - defined as part of IoT SDK
- software.amazon.iot.thingtype.ThingTypeProperties - defined as part of CFN resource modeling 

As part of the Translator, we are basically converting the CFN model to IoT SDK model and vice-versa